packages/banner/src/components/LidoStakeBanner.tsx

```diff
@@ -37,7 +37,7 @@ const LidoStakeBanner: React.FC<Props> = (props) => {
   const defiLink = getDefiLink(referrerId);
   const defiApy = getDefiApy();
 
-  const [apy, setApy] = useState(STATIC_DEFAULT_APY);
+  const [apy, setApy] = useState<string | number>(STATIC_DEFAULT_APY);
```

Comment:
why string | number?

Reply:
because of max.apy.**toFixed(2)** (toFixed returns string)

of course, we can add +, or `toNumber()`. But it's not necessary, because React will convert it to string again during rendering. So it's just typings, not requirement to make them similar.

Reply:
In some case it would be a bug in future. Better use +max.apy.toFixed(2) and strict number typing

Reply:
I don't think so, it's too primitive case, both types are ok. But okay, let it be.

Reply:
done